hail_scripts/reference_data/config.py

```diff
@@ -102,31 +112,38 @@ def custom_gnomad_select_v3(ht):
         '37': {
             'path': 'gs://seqr-reference-data/GRCh37/dbNSFP/v2.9.3/dbNSFP2.9.3_variant.ht',
             'select': [
-                'SIFT_pred',
-                'Polyphen2_HVAR_pred',
-                'MutationTaster_pred',
-                'FATHMM_pred',
                 'MetaSVM_pred',
                 'REVEL_score',
                 'GERP_RS',
                 'phastCons100way_vertebrate',
             ],
+            'custom_select': dbnsfp_custom_select,
```

Comment:
A few questions here:
- `REVEL_score` is a string that we parse to a float [here](https://github.com/broadinstitute/seqr/compare/hail-backend#diff-8fc83668e54552fcb504d1667e3256c36da1ceb8c893b64fad62c2dc0d508479R141).  Should it just be a `float`? 
- `MetaSVM_pred` appears to be an enum, should we also map it?
```
In [10]: ht.aggregate(hl.agg.counter(ht.MetaSVM_pred))
Out[10]: {'D': 13663364, 'T': 68434853, None: 7519568}
```

Reply:
yes lets map it